src/main_loop.rs

```diff
@@ -24,6 +24,9 @@ use std::{io, rc::Rc};
 use tokio_core::reactor::Handle;
 use tokio_io::IoStream;
 
+#[cfg(feature = "dbus_mpris")]
+use crate::dbus_mpris::{ChangedTrack, DbusServer, TrackChange};
```

Comment:
I tried building this with `cargo build --release --all-features` (rustc 1.53.0), but it complained:

```
error[E0252]: the name `DbusServer` is defined multiple times
  --> src/main_loop.rs:28:39
```

…and

```
warning: unused import: `DbusServer`
  --> src/main_loop.rs:28:39
```

Am I missing something?